Content.Server/CartridgeLoader/Cartridges/LogProbeCartridgeSystem.cs

```diff
@@ -63,9 +82,49 @@ private void OnUiReady(Entity<LogProbeCartridgeComponent> ent, ref CartridgeUiRe
         UpdateUiState(ent, args.Loader);
     }
 
+    private void OnMessage(Entity<LogProbeCartridgeComponent> ent, ref CartridgeMessageEvent args)
+    {
+        if (args is LogProbePrintMessage cast)
+            PrintLogs(ent, cast.User);
+    }
+
+    private void PrintLogs(Entity<LogProbeCartridgeComponent> ent, EntityUid user)
+    {
+        if (string.IsNullOrEmpty(ent.Comp.EntityName))
+            return;
+
+        if (_timing.CurTime < ent.Comp.NextPrintAllowed)
+            return;
+
+        ent.Comp.NextPrintAllowed = _timing.CurTime + ent.Comp.PrintCooldown;
+
+        var paper = Spawn(ent.Comp.PaperPrototype, _transform.GetMapCoordinates(user));
+        _label.Label(paper, ent.Comp.EntityName); // label it for easy identification
```

Comment:
If the label is just `ent.Comp.EntityName`, does that mean probing like `Chief Engineer's Locker` will make the printout's label just be `paper (Chief Engineer's Locker)`?
The forensic scanner appends something to the label, so this ought to do the same.

Reply:
the prefix is still the papers name in the prototype not just "paper"